src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -182,7 +258,7 @@ const CooperationDetails = () => {
   return (
     <PageWrapper>
       <Box sx={styles.header}>
-        <StatusChip status={cooperationStatus} />
+        <StatusChip status={cooperationStatusChip} />
```

Comment:
Rename it back, please

```suggestion
        <StatusChip status={cooperationStatus} />
```